A32NX/html_ui/Pages/A32NX_Core/A32NX_BaroSelector.js

```diff
@@ -0,0 +1,50 @@
+// Used to determine if an ICAO code uses InHG or HPa.
+function AirportUsesHPA(icao) {
+    const InHgRegions = ["K", "C", "M", "P", "RJ", "RO", "TI", "TJ"];
+    for (let i = 0; i < InHgRegions.length; i++)
+        if (icao.substr(0, InHgRegions[i].length) == InHgRegions[i]) return false;
+    return true;
+}
```

Comment:
Should start with a lowercase character and be inside the class (move inside class, call using `this` and remove `function` keyword).

Reply:
I'd like to be able to use this function in the MCDU options page as well (A320_Neo_CDU_OPTIONS_Menu.js). It'd be nice to be able to AUTO set when the option is toggled as well. Any way of doing that without duplicating code? Had it as a function of its own to remind me of that 😉